versions/src/main/resources/mixins.oneconfig.json

```diff
@@ -1,22 +1,23 @@
 {
-    "compatibilityLevel": "JAVA_8",
```

Comment:
why was the formatting here changed? hard 4 tabs are the default? doesnt really matter since in v1, formatting will be standardized (pls see the thing in higher i sent about this).

Reply:
idk lol